src/worker.py

```
@@ -1329,6 +1314,331 @@ async def serve_static(path: str, env):
+                "display_name":   info["display_name"],
+                "text":           text,
+                "timestamp":      data.get("timestamp", ""),
+            }))
+
+        elif msg_type == "update_seat":
+            seat_id = data.get("seat_id", "")
+            # Validate seat_id: must be "seat-N" where N is 1..DESK_ROWS*DESK_COLS
+            if not isinstance(seat_id, str) or not re.fullmatch(r"seat-\d+", seat_id):
+                return
+            seat_num = int(seat_id.split("-")[1])
+            if seat_num < 1 or seat_num > 15:  # 3 rows × 5 cols = 15 seats
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)